lib/web_ui/lib/src/engine/host_node.dart

```diff
@@ -111,6 +111,19 @@ class ShadowDomHostNode implements HostNode {
       browserEngine: browserEngine,
       hasAutofillOverlay: browserHasAutofillOverlay(),
     );
+
+    // Removes password reveal icon for text inputs in Edge browsers.
+    // Style tag w/styles needs to be injected into DOM because non-Edge
```

Comment:
nit: "Style tag w/styles" sounds a bit ambiguous. Maybe rewording it and using the full word "with" instead of "w/" would help?